### PR TITLE
[docs] bm added a link to the node-manager module

### PR DESCRIPTION
**Pull request by alexey-gavrilov-flant**
## Description
added a link to the node-manager module

## Why do we need it, and what problem does it solve?
added a link to the node-manager module so that you can immediately read how the group concept works. 

## Checklist
- [ ] The code is covered by unit tests.
- [ ] e2e tests passed.
- [ ] Documentation updated according to the changes.
- [ ] Changes were tested in the Kubernetes cluster manually.

## Changelog entries

```changes
section: docs
summary: bm added a link to the node-manager module
impact_level: default
```

### CODE REVIEW

**Comment by deckhouse-BOaTswain** [comment]
Aye, aye, @z9r5. I've started the workflow for label 'deploy/web/test'...

File: docs/site/_includes/getting_started/bm/STEP_FINALIZE.md [hunk 1]
```diff
@@ -4,7 +4,7 @@
 
 At this point, you have created a basic **single-master** cluster.
 
-For real-world conditions (production and test environments), you need to add additional nodes to the cluster according to <a href="/documentation/latest/modules/040-node-manager/faq.html#how-do-i-add-a-static-node-to-a-cluster">the documentation</a>.
+For real-world conditions (production and test environments), you need to <a href="/documentation/latest/modules/040-node-manager/faq.html#how-do-i-add-a-static-node-to-a-cluster">add nodes</a> to the cluster and familiarize yourself with <a href="/documentation/latest/modules/040-node-manager/">the Node Management</a> module.
```
Comment (z9r5) [inline docs/site/_includes/getting_started/bm/STEP_FINALIZE.md:7]:
```suggestion
For real-world conditions (production and test environments), <a href="/documentation/latest/modules/040-node-manager/faq.html#how-do-i-add-a-static-node-to-a-cluster">add nodes</a> to the cluster and familiarize yourself with how to <a href="/documentation/latest/modules/040-node-manager/">manage nodes</a>.
```

File: docs/site/_includes/getting_started/bm/STEP_FINALIZE_RU.md [hunk 2]
```diff
@@ -4,7 +4,7 @@
 
 На данном этапе вы создали кластер, который состоит из **единственного** master-узла.
 
-Для полноценной работы кластера добавьте узлы, согласно <a href="/documentation/latest/modules/040-node-manager/faq.html#как-добавить-статичный-узел-в-кластер">документации</a> (рекомендуется для production-окружений и тестовых сред).
+Для полноценной работы кластера <a href="/documentation/latest/modules/040-node-manager/faq.html#как-добавить-статичный-узел-в-кластер">добавьте узлы</a> в кластер и ознакомьтесь с модулем <a href="/documentation/latest/modules/040-node-manager/">Управление узлами</a> (рекомендуется для production-окружений и тестовых сред).
```
Comment (z9r5) [inline docs/site/_includes/getting_started/bm/STEP_FINALIZE_RU.md:7]:
```suggestion
Для полноценной работы кластера <a href="/documentation/latest/modules/040-node-manager/faq.html#как-добавить-статичный-узел-в-кластер">добавьте узлы</a> в кластер и ознакомьтесь с <a href="/documentation/latest/modules/040-node-manager/">управлением узлами</a> (рекомендуется для production-окружений и тестовых сред).
```

**Comment by deckhouse-BOaTswain** [comment]
Aye, aye, @z9r5. I've started the workflow for label 'deploy/web/test'...

File: docs/site/_includes/getting_started/bm/STEP_FINALIZE_RU.md [hunk 3]
```diff
@@ -4,7 +4,7 @@
 
 На данном этапе вы создали кластер, который состоит из **единственного** master-узла.
 
-Для полноценной работы кластера добавьте узлы, согласно <a href="/documentation/latest/modules/040-node-manager/faq.html#как-добавить-статичный-узел-в-кластер">документации</a> (рекомендуется для production-окружений и тестовых сред).
+Для полноценной работы кластера <a href="/documentation/latest/modules/040-node-manager/faq.html#как-добавить-статичный-узел-в-кластер">добавьте узлы</a> в кластер и ознакомьтесь с <a href="/documentation/latest/modules/040-node-manager/">управлением узлами</a> (рекомендуется для production-окружений и тестовых сред).
```
Comment (z9r5) [inline docs/site/_includes/getting_started/bm/STEP_FINALIZE_RU.md:7]:
```suggestion
Для полноценной работы кластера <a href="/documentation/v1/modules/040-node-manager/faq.html#как-добавить-статичный-узел-в-кластер">добавьте узлы</a> в кластер и ознакомьтесь с <a href="/documentation/v1/modules/040-node-manager/">управлением узлами</a> (рекомендуется для production-окружений и тестовых сред).
```

File: docs/site/_includes/getting_started/bm/STEP_FINALIZE.md [hunk 4]
```diff
@@ -4,7 +4,7 @@
 
 At this point, you have created a basic **single-master** cluster.
 
-For real-world conditions (production and test environments), you need to add additional nodes to the cluster according to <a href="/documentation/latest/modules/040-node-manager/faq.html#how-do-i-add-a-static-node-to-a-cluster">the documentation</a>.
+For real-world conditions (production and test environments), <a href="/documentation/latest/modules/040-node-manager/faq.html#how-do-i-add-a-static-node-to-a-cluster">add nodes</a> to the cluster and familiarize yourself with how to <a href="/documentation/latest/modules/040-node-manager/">manage nodes</a>.
```
Comment (z9r5) [inline docs/site/_includes/getting_started/bm/STEP_FINALIZE.md:7]:
```suggestion
For real-world conditions (production and test environments), <a href="/documentation/v1/modules/040-node-manager/faq.html#how-do-i-add-a-static-node-to-a-cluster">add nodes</a> to the cluster and familiarize yourself with how to <a href="/documentation/v1/modules/040-node-manager/">manage nodes</a>.
```

**Comment by deckhouse-BOaTswain** [comment]
Aye, aye, @z9r5. I've started the workflow for label 'deploy/web/test'...

**Pull request merged**